<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GUIClass</class>
 <widget class="QMainWindow" name="GUIClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1366</width>
    <height>768</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>GUI</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QTabWidget" name="mainTab">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="mainTab_Webcams">
       <attribute name="title">
        <string>Webcams</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="2" column="0">
         <widget class="QTabWidget" name="leftTab">
          <property name="currentIndex">
           <number>0</number>
          </property>
          <widget class="QWidget" name="leftTab_RGB">
           <attribute name="title">
            <string>RGB</string>
           </attribute>
           <layout class="QGridLayout" name="gridLayout_3">
            <item row="0" column="0">
             <widget class="QLabel" name="leftRGB">
              <property name="text">
               <string>TextLabel</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <layout class="QHBoxLayout" name="horizontalLayout_2">
              <item>
               <widget class="QCheckBox" name="leftArenaCheck">
                <property name="text">
                 <string>Arena</string>
                </property>
                <property name="checked">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QCheckBox" name="leftBallsCheck">
                <property name="text">
                 <string>Tennis Balls</string>
                </property>
                <property name="checked">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QCheckBox" name="leftObstaclesCheck">
                <property name="text">
                 <string>Obstacles</string>
                </property>
                <property name="checked">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QCheckBox" name="leftRobotCheck">
                <property name="text">
                 <string>Robot</string>
                </property>
                <property name="checked">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="leftTab_HSV">
           <attribute name="title">
            <string>HSV</string>
           </attribute>
           <layout class="QGridLayout" name="gridLayout_6">
            <item row="0" column="0">
             <widget class="QLabel" name="leftHSV">
              <property name="text">
               <string>TextLabel</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="leftTab_Threshold">
           <attribute name="title">
            <string>Threshold</string>
           </attribute>
           <layout class="QGridLayout" name="gridLayout_7">
            <item row="0" column="0">
             <widget class="QLabel" name="leftThreshold">
              <property name="text">
               <string>TextLabel</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <layout class="QHBoxLayout" name="leftRadios">
              <item>
               <widget class="QRadioButton" name="leftArenaRadio">
                <property name="text">
                 <string>Arena</string>
                </property>
                <property name="checkable">
                 <bool>true</bool>
                </property>
                <property name="checked">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QRadioButton" name="leftBallsRadio">
                <property name="text">
                 <string>Tennis Balls</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QRadioButton" name="leftObstaclesRadio">
                <property name="text">
                 <string>Obstacles</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QRadioButton" name="leftRobotRadio">
                <property name="text">
                 <string>Robot</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QTabWidget" name="rightTab">
          <property name="currentIndex">
           <number>0</number>
          </property>
          <widget class="QWidget" name="rightTab_RGB">
           <attribute name="title">
            <string>RGB</string>
           </attribute>
           <layout class="QGridLayout" name="gridLayout_4">
            <item row="0" column="0">
             <widget class="QLabel" name="rightRGB">
              <property name="text">
               <string>TextLabel</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <layout class="QHBoxLayout" name="horizontalLayout_3">
              <item>
               <widget class="QCheckBox" name="rightArenaCheck">
                <property name="text">
                 <string>Arena</string>
                </property>
                <property name="checked">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QCheckBox" name="rightBallsCheck">
                <property name="text">
                 <string>Tennis Balls</string>
                </property>
                <property name="checked">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QCheckBox" name="rightObstaclesCheck">
                <property name="text">
                 <string>Obstacles</string>
                </property>
                <property name="checked">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QCheckBox" name="rightRobotCheck">
                <property name="text">
                 <string>Robot</string>
                </property>
                <property name="checked">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="rightTab_HSV">
           <attribute name="title">
            <string>HSV</string>
           </attribute>
           <layout class="QGridLayout" name="gridLayout_8">
            <item row="0" column="0">
             <widget class="QLabel" name="rightHSV">
              <property name="text">
               <string>TextLabel</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="rightTab_Threshold">
           <attribute name="title">
            <string>Threshold</string>
           </attribute>
           <layout class="QGridLayout" name="gridLayout_9">
            <item row="0" column="0">
             <widget class="QLabel" name="rightThreshold">
              <property name="text">
               <string>TextLabel</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <layout class="QHBoxLayout" name="right">
              <item>
               <widget class="QRadioButton" name="rightArenaRadio">
                <property name="text">
                 <string>Arena</string>
                </property>
                <property name="checkable">
                 <bool>true</bool>
                </property>
                <property name="checked">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QRadioButton" name="rightBallsRadio">
                <property name="text">
                 <string>Tennis Balls</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QRadioButton" name="rightObstaclesRadio">
                <property name="text">
                 <string>Obstacles</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QRadioButton" name="rightRobotRadio">
                <property name="text">
                 <string>Robot</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QPushButton" name="rightCalibrate">
          <property name="text">
           <string>Calibrate</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QPushButton" name="leftCalibrate">
          <property name="text">
           <string>Calibrate</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QPushButton" name="leftReset">
          <property name="text">
           <string>Reset</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QPushButton" name="rightReset">
          <property name="text">
           <string>Reset</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="mainTab_TopView">
       <attribute name="title">
        <string>Top View</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_5">
        <item row="0" column="2">
         <widget class="QLabel" name="topView">
          <property name="text">
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="leftFinal">
          <property name="text">
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLabel" name="rightFinal">
          <property name="text">
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="gui.qrc"/>
 </resources>
 <connections/>
</ui>
